app/data/census_household_composition.json

```diff
@@ -0,0 +1,100 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "questionnaire_id": "23",
+    "schema_version": "0.0.1",
+    "survey_id": "023",
+    "title": "Census Theme",
+    "description": "",
+    "theme": "census",
+    "introduction": {
+        "description": "",
+        "information_to_provide": []
+    },
+    "display": {
+        "properties": {}
+    },
+    "eq_id": "1",
+    "groups": [
+        {
+            "blocks": [
+
+                {
+                    "display": {
+                        "properties": {}
+                    },
+                    "id": "block",
+                    "sections": [
+                        {
+                            "display": {
+                                "properties": {}
+                            },
+                            "id": "section",
+                            "questions": [
+                                {
+                                  "description": "",
+                                  "display": {
+                                      "properties": {}
+                                  },
+                                  "id": "household-question",
+                                  "title": "Who usually lives here?",
+                                  "description": "<br> <div> <h3>Include:</h3> <ul> <li>Yourself, if this is your permanent or family home </li> <li>Family members including partners, children and babies born on or before 9 April 2017</li> <li>Students and/or school children who live away from home during term time</li> <li>Housemates, tenants or lodgers</li> </ul> </div>",
+                                  "type": "Household",
+                                  "validation": [],
+                                  "answers": [
+                                    {
+                                        "id": "person",
+                                        "type": "Composite",
+                                        "q_code": "9999",
+                                        "mandatory": false,
+                                        "answers": [
```

Comment:
Maybe we should call these something else from answers, maybe something like parts or something?